Automattic/wp-calypso
# Help-center: enable email support form

#### Changes proposed in this Pull Request

* Connect the email support form to the API using the `useSubmitTicketMutation` hook. 
* Connect the success state to the success screen.

#### Testing instructions

1. Go to the help-center.
2. Pick "Email support".
3. Fill the form.
4. Send. 
5. The loading state should disable the CTA and the back button. 
6. You should see the success screen.
